configs/tasks/rearrangepick_replica_cad.yaml

```
@@ -49,6 +44,11 @@ TASK:
     LOCAL_SENSOR:
         TYPE: "LocalizationSensor"
     SENSORS: ["TARGET_START_SENSOR", "GOAL_SENSOR", "JOINT_SENSOR", "IS_HOLDING_SENSOR", "EE_SENSOR", "LOCAL_SENSOR"]
+    ROBO_FORCE:
+        TYPE: "RoboForce"
+        MIN_FORCE: 20.0
+    ROBO_COLLS:
```

Let's have full names for the sensors and variables to make it self descriptive:
```suggestion
    ROBOT_COLLISIONS:
```

---

Fixed all instances of ROBO instead of ROBOT.